.github/workflows/ci.yml

```diff
@@ -1,6 +1,9 @@
 name: CI
 
 on:
+  push:
+    paths:
+    - '**.ts'
```

Comment:
Can you clarify the intention of this change? 

From my understanding (which is admittedly not great), this will run the tests on any push (to any branch? Not just open PR branches?) that contains a change to a typescript file.  It won't affect the or trigger defined below.

I believe `paths` is also valid in `pull_request` triggers, but I'm not sure why you would want to filter the whole pipeline to just typescript file changes?

Reply:
yes that's correct (my understanding isnt great either).
because the repo files is in `.ts` and from my understanding if we made a PR to change `.md` file the action will run. so instead of excluding the other files, set the path to .ts made more sense. (hopefully I didnt missread the repo)

Reply:
Given the discord discussion, I  don't think we need this. The test job looks ok though.

Remove these lines and we'll see if the tests pass the checks or not.

Reply:
@nobrayner your thoughts, otherwise I will push and possibly fix later

Reply:
If you want to keep the paths - which does work for the pull_request event - then I would suggest any `.ts|.js|.json` files. That way any config changes would trigger a CI run as well.

Really though, you are better off setting an `ignore-paths` and going from there. I vote getting rid of it :grimacing: